source/_components/foursquare.markdown

```diff
@@ -28,6 +28,23 @@ Configuration variables:
 - **access_token** (*Required*): A Foursquare API access token.
 - **push_secret** (*Required*): The push secret that Foursquare provides to you in the app dashboard.
 
+#### Getting the access token ####
+
+After you have registered your APP on your [My Apps Page](https://foursquare.com/developers/apps) you get a `CLIENT_ID` and you have specified a
+`REDIRECT_URL` which can be any URL you like, but since it will get your access token via a HTTP GET request, it should be a URL which will ignore the `access_token` HTTP GET variable. 
+Visit the following URL in your browser:
+
+```
+https://foursquare.com/oauth2/authenticate
```

Comment:
Should these be on separate lines? A tad easier to read but might be harder for a new reader.